.github/workflows/test.yml

```diff
@@ -52,9 +52,9 @@ jobs:
       id: go
 
     - name: Verify
-      uses: golangci/golangci-lint-action@v4
+      uses: golangci/golangci-lint-action@v6
       with:
-        version: v1.56.1
+        version: v1.59.1
```

Comment:
however , when running things locally for me I use the make lint which is the hack/lint.sh file that grabs version locally pinned on that file -> I think we should update that also to 1.56
but my real doubt is that file is also running:
```
$1 run --security-opt label=disable --rm \
  -v  ${HOME}/.cache/golangci-lint:/cache -e GOLANGCI_LINT_CACHE=/cache \
  -v $(pwd):/app -w /app -e GO111MODULE=on docker.io/golangci/golangci-lint:${VERSION} \
	golangci-lint run --verbose --print-resources-usage \
	--modules-download-mode=vendor --timeout=15m0s && \
```
which should use the same config file right? so someone before pushing the PR if they run make lint will get same results as CI Lint lane correct?

Reply:
Everything uses the same config file. My changes did not really have an impact on which config file is being used.